browser/src/control/Control.NotebookbarImpress.js

```diff
@@ -1555,6 +1555,14 @@ L.Control.NotebookbarImpress = L.Control.NotebookbarWriter.extend({
 
 	getLayoutTab: function() {
 		var content = [
+			{
+				'id': 'slide-size:SlideSizeMenu',
+				'class': 'unoSlideSize',
+				'type': 'exportmenubutton',
```

Comment:
This is custom dropdown only for "export options" please do not use it here.
Please use regular "menubutton".

Reply:
Got it, though we had thought about having icons on the submenu dropdown of Slide Size

cc @pedropintosilva

Reply:
we can add them to the regular menu if support is missing (I don't remember)